test/Microsoft.AspNet.Mvc.FunctionalTests/SerializableErrorTests.cs

```diff
@@ -43,7 +51,9 @@ public class SerializableErrorTests
             XmlAssert.Equal(expectedXml, responseData);
         }
 
-        [Theory]
+        [ConditionalTheory]
+        // Mono issue - https://github.com/aspnet/External/issues/18
+        [FrameworkSkipCondition(RuntimeFrameworks.Mono)]
```

Comment:
Make this MemberData (since data is the same for all these tests). DCS should be the only one disabled in mono.